apps/demo/app/custom-ui/[...puckPath]/client.tsx

```diff
@@ -281,13 +281,15 @@ const CustomDrawer = () => {
   const { getPermissions } = usePuck();
 
   return (
-    <Drawer direction="horizontal">
+    <Drawer>
       <div
         style={{
-          display: "flex",
+          display: "grid",
```

Comment:
[maybe consider adding](https://css-tricks.com/snippets/css/complete-guide-grid/#aa-grid-auto-flow)
`grid-auto-flow: column;`

Reply:
What would the intention of adding that be?